nixos/modules/services/networking/ntp/chrony.nix

```diff
@@ -55,14 +58,16 @@ in
         type = types.listOf types.str;
         description = ''
           The set of NTP servers from which to synchronise.
+
+          Can add options by appending them at the end of the string
         '';
       };
 
       serverOption = mkOption {
         default = "iburst";
         type = types.enum [ "iburst" "offline" ];
```

Comment:
`serverOption` should be a list of strings since Chrony has 32 options for the `server` plus another one for the `pool`. enum is unsuitable for this option because some `serverOption` have arguments such as `minpoll $interval` and a server directive can have multiple serverOption.